front-end-101/final_project_plan/style.css

```diff
@@ -229,6 +251,25 @@ form {
 }
 
 
+@media (max-width: 768px) {
+
+    /* Для экранов шириной <= 768px */
+    #ent_block1,
+    #ent_block2,
+    #ent_block3,
+    #ent_block4 {
+        flex: none;
+        width: 100%;
+
+    }
+
+    #entertainment {
+        display: flex;
+        flex-direction: column;
+        align-items: stretch;
+    }
```

Comment:
all of this can be and should be under the asme media query that you defined above